step.yml

```diff
@@ -25,6 +25,11 @@ inputs:
       summary: "The root dir of your Flutter project."
       description: "The root dir of your Flutter project."
       is_required: true
+  - bitrise_test_result_dir: $BITRISE_TEST_RESULT_DIR
+    opts:
+      title: Step's test result directory
+      description: Root directory for all test results created by the Bitrise CLI
+      is_required: true
```

Comment:
Please add the property: `is_dont_change_value: true`, so the field is uneditable on the Workflow Editor UI.

Reply:
we did not make this read-only for the custom test results export step, could there be a scenario where the user needs to change this?

Reply:
If they really need to change it, they can in the bitrise.yml directly. This just disables the input box on the UI.

Reply:
done